### PR TITLE
Get single profile picture endpoint

### PR DESCRIPTION
**Pull request by lazynina**
add get single profile picture endpoint, remove profile pic from profile entry response to lighten payloads

### CODE REVIEW

File: routes/user.go [hunk 1]
```diff
@@ -928,6 +916,55 @@ func (fes *APIServer) augmentProfileEntry(
...
+	}
+	publicKeyBytes, _, err := lib.Base58CheckDecode(publicKeyBase58Check)
+	if err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf("GetSingleProfilePicture: Problem decoding user public key: %v", err))
+		return
+	}
+	// We ignore errors here and simply return
+	profilePicture, _ := fes._getProfilePictureForPublicKey(publicKeyBytes)
+
+	if err = json.NewEncoder(ww).Encode(GetSingleProfilePictureResponse{
```
Comment (maebeam) [inline routes/user.go:960]: We should just serve the image itself as bytes so browsers recognize the MIME type and we get free client and edge caching

Comment (maebeam) [reply routes/user.go:960]: If no image, return a 404 and let the frontend use the default_profile_pic

**Review by maebeam — APPROVED** [review]

**Comment by harpreet9629** [comment]
How we can get single profile picture with api?

**Comment by lazynina** [comment]
> How we can get single profile picture with api?

GET /api/v0/get-single-profile-picture/{PublicKeyBase58Check}

^thats the endpoint

**Comment by harpreet9629** [comment]
I m getting error on send bitclout
Error: SendBitclout: User with name not found!

I m using public key for both sender and receiver

**Pull request merged**